pages/linux/pkgadd.md

```diff
@@ -0,0 +1,15 @@
+# pkgadd
+
+> Add a package to a CRUX system
+
+- install a local software package
+
+`pkgadd {{package-name}}`
+
+- update a already installed package from a local package
+
+`pkgadd -u {{package-name}}`
+
+- force installation of a package
```

Comment:
Is this one a very useful or common example? If something doesn't work and I feel it should, I just add `-f` to whatever command. It feels like one of those flags like `--version` or `--help`.